source/docs/software/wpilib-tools/axon/index.rst

```diff
@@ -0,0 +1,23 @@
+:orphan:
+
+Axon
+====
+
+Axon is a tool to simply train machine learning models.
```

Comment:
This needs to be much more informative. It trains learning models using what, how, and why? What usage is this for FIRST? Give a paragraph response advertising ;)

Reply:
I will probably rewrite this.